examples/3.tutorials/lit-05-SimpleMotion.jl

```diff
@@ -4,23 +4,19 @@ using KomaMRI # hide
 sys = Scanner() # hide
 
 # It can also be interesting to see the effect of the patient's motion during an MRI scan.
-# For this, Koma provides the ability to add `motion <: MotionModel` to the phantom.
-# In this tutorial, we will show how to add a [`SimpleMotion`](@ref) model to a 2D brain phantom.
+# For this, Koma provides the ability to add `motion <: AbstractMotionSet` to the phantom.
```

Comment:
`motion <: AbstractMotionSet` I thought we changed `AbstractMotionSet` to `AbstractMotion`. Set has a precise mathematical meaning.

`AbstractMotion`:
- `struct NoMotion`
- `struct MotionList` (The same as an AbstractArray{Motion} but we didn't use arrays, because ... ? we will need to discuss this in the paper)
	- `struct Motion` (This one makes sense)
		- `Action <: AbstractAction`
		- `TimeSpan <: AbstractTimeSpan`
		- `SpinSpan <: AbstractSpinSpan` (Is the user meant to extend this one?)
		
Is it like this? 

What function must I define to generate a new `AbstractMotion`/`AbstractAction`/...?

If I want to define my new custom motion, should create a new AbstractAction or a new AbstractMotion? 

Documenting all of this is important, let's generate an issue.

Reply:
I change `AbstractMotionSet` to `AbstractMotion` and put your message in issue https://github.com/JuliaHealth/KomaMRI.jl/issues/395

Reply:
Ok thanks

> If I don't want to define a list, can I just define obj.motion = Translate(...)?

It would be nice if we could do this (motion without list)..maybe this can be a different issue.